src/shared/containers/VideoPluginContainer.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { Resource } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+import ReactPlayer from 'react-player';
+import * as moment from 'moment';
+import { Collapse, Modal, Button, List, Avatar } from 'antd';
+
+import '../styles/video-plugin.less';
+
+const { Panel } = Collapse;
+
+type VideoProps = {
+  orgLabel: string;
+  projectLabel: string;
+  resource: Resource;
+};
+
+type VideoObject = {
+  name: String;
+  description: String;
+  thumbnailUrl: [];
+  uploadDate: Date;
+  duration: String;
+  embedUrl: String;
+};
+
+const VideoPluginContainer: React.FunctionComponent<VideoProps> = ({
+  resource,
+  orgLabel,
+  projectLabel,
+}) => {
+  const nexus = useNexusContext();
+  const [videoData, setVideoData] = React.useState<any>();
+  const [isModalVisible, setIsModalVisible] = React.useState<boolean>(false);
+  const [selectedVideo, setSelectedVideo] = React.useState<any>();
+
+  const handleOk = () => {
+    const isModalVisible = false;
+    setIsModalVisible(isModalVisible);
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+    setSelectedVideo(null);
+  };
+
+  React.useEffect(() => {
+    loadVideo();
+  }, []);
+
+  const handleSelectedVideo = (video: any) => {
+    const selectedVideo = video;
+    setSelectedVideo(selectedVideo);
+    setIsModalVisible(true);
+  };
+  const loadVideo = async () => {
+    const videoResource = (await nexus.Resource.get(
+      orgLabel,
+      projectLabel,
+      encodeURIComponent(resource['@id'])
+    )) as Resource<{
+      video: VideoObject[];
+    }>;
+    const videoData = videoResource['video']
```

Comment:
Can we use dot notation everywhere? Like `videoResource.video`.
Dot notation is more performant and easier to read. There is no need to use bracket notation as there there are no special character in the property name.